api/src/main/java/com/vimeo/networking2/adapters/VimeoCallback.kt

```diff
@@ -0,0 +1,40 @@
+package com.vimeo.networking2.adapters
```

Comment:
is the `adapters` package the correct package for this? could it be moved into the parent package as that will be more accessible for consumers. perhaps we could follow what some other libraries do where they have an `internal` package where we could put the internal classes in the adapter. happy to talk offline about the package structure.